wiki/Getting_Started_Contributing_to_Wiki/en.md

```diff
@@ -0,0 +1,178 @@
+- A markdown previewer with **GFM** Support (Github-Flavoured Markdown), we use [Shiba](https://github.com/rhysd/Shiba) or Atom and Visual Studio Code's Built In Markdown Previewers (If you plan to make changes locally).
+
+## Creating a GitHub Account 
+ To get started in contributing to osu!wiki, you must first sign up for a GitHub Account.
+
+ Go to the [GitHub Homepage](https://github.com).
+
+ ![GitHub Homepage Screenshot](1.JPG "Github Homepage")
+
+ Then click "Sign Up" and fill the following Fields like this example.
+
+ ![Registration Snapshot Example](2.jpg "Registration Snapshot")
+
+ Now click "Create an account" and you're well on your way to contribute to osu!wiki.
+
+
+## Forking
+ 
+ To be able to make changes to the repository, you must first create your own copy of the repository. This is known as "Forking".
+ 
+ Go to the [GitHub repository](https://github.com/ppy/osu-wiki).
+ 
+ You should see something like this.
+
+![osu-wiki Repository Snapshot](3.JPG "Snapshot of the osu!Wiki Repository")
+
+Now click the "Fork" Button on the top-right and it would prompt you on where to fork the repository.
+
+![Fork prompt dialogue](4.JPG "Fork prompt dialogue")
+
+If you have already made a fork, the Prompt will also say where was it forked.
+
+Now click at your profile and GitHub would redirect you to where the fork is.
+
+And you should see something like this.
+
+![Fork Snapshot](5.JPG "Fork Snapshot")
+
+Congratulations! You made a copy of the osu!wiki repository! You can now proceed in making changes.
+
+## Making Changes 
+
+### Through GitHub's Web Editor
+If you don't want the hassle of cloning your fork into your PC, GitHub offers Editing through your browser.
+
+Go to your desired Folder that you want to edit and pick a Markdown File to edit.
+
+_Note: Each article is grouped into folders and each folder contains markdown files arranged by language and their media.
+Please follow this format if you're creating a new Article._
+
+Now click your desired Markdown file to edit. And you should see something like this.
+
+![GitHub Preview](7.JPG "Github Preview")
+
+Now Click on the Pencil Icon to edit. And it should redirect you to the Web Editor.
+
+![GitHub's Web Editor](8.JPG "web Editor Snapshot")
+
+If you want to preview your changes, click the "Preview Changes" Tab. Click "Edit File" to go back to Editing.
+
+After making your changes, scroll down and Fill the following fields to explain your edit to other contributors.
+
+![Commit Message](9.JPG "Commit Message")
+
+You would be given a option where to save your edits.
+
+- "Commit Directly to the ``xy`` branch" will commit the changes through your branch directly.
+
+- "Create a New Branch" will automatically create a Branch for your edits. This is useful if you're making bulk edits.
+
+Pick a option and click "Commit Changes".
+
+### Through your local Machine
+
+GitHub under the hood is a Git Service. You can clone your fork in your Machine and make changes there.
```

Comment:
I think it would be best also for the people to get familiarized with the Git CLI since they may need it just in case they want to do a rebase and such.